modules/nw-network-observability-operator.adoc

```diff
@@ -62,17 +62,16 @@ netobserv-ebpf-agent-xf5jh   1/1     Running   0          151m
 
 `netobserv-ebpf-agent` pods monitor network interfaces of the nodes to get flows and send them to `flowlogs-pipeline` pods.
 
-. If you are using the {loki-op}, check the status of pods running in the `openshift-operators-redhat` namespace by entering the following command:
+. If you are using the {loki-op}, check the status of the `LokiStack` custom resource `component` pods running in the `netobserv` namespace by entering the following command:
```

Comment:
does this mean "check the status of the `component` pods of `LokiStack` custom resource"? If yes, it can be rewritten. Also if component isn't a term used in code or name of the pod, backticks can be removed.